spec/support/shared/shopify_shop.rb

```diff
@@ -0,0 +1,20 @@
+RSpec.shared_context 'shopify_shop' do
+  let(:refund_amount) { 100 }
+  let(:shopify_payment_method) do
+    gateway = Spree::Gateway::ShopifyGateway.new
+    gateway.set_preference(:api_key, ENV.fetch('SHOPIFY_API_KEY'))
+    gateway.set_preference(:password, ENV.fetch('SHOPIFY_PASSWORD'))
+    gateway.set_preference(:shop_name, ENV.fetch('SHOPIFY_SHOP_NAME'))
+    gateway
+  end
+
+  def mock_request(endpoint, extension)
```

Comment:
Hmm @cabouffard and @DanielWright this resolves just fine. It creates the correct webmock, yadda yadda. However, I then want to invoke this mock by calling `ShopifyAPI::Order.find(order_numb)` but this fails with the following:

```
NoMethodError: undefined method `path' for nil:NilClass
from /Users/braidn/src/glossier-retail/.bundle/ruby/2.3.0/gems/activeresource-4.1.0/lib/active_resource/base.rb:688:in `prefix'
```

This occurs regardless of calling the mock_request method so it seems like it's a deeper issue. Any ideas would be amaze. Thanks.